lib/beaker/host/pswindows/pkg.rb

```diff
@@ -27,17 +27,60 @@ def uninstall_package(_name, _cmdline_args = '')
   # Examine the host system to determine the architecture, overrides default host determine_if_x86_64 so that wmic is used
   # @return [Boolean] true if x86_64, false otherwise
   def determine_if_x86_64
-    identify_windows_architecture.include?('64')
+    identify_windows_architecture.include?('AMD64')
   end
 
   private
 
   # @api private
   def identify_windows_architecture
+    arch = nil
+    execute('echo %PROCESSOR_ARCHITECTURE%', :accept_all_exit_codes => true) do |result|
+      arch = if result.exit_code == 0
+               result.stdout.strip
+             else
+               identify_windows_architecture_wmic
+             end
+    end
+
+    arch = 'AMD64' if arch.include?('EM64T') # Normalize Intel 64-bit architecture name
```

Comment:
The documentation said that was only for Windows XP so I'm not sure we need to include this.